qiskit_experiments/calibration/calibrations.py

```diff
@@ -897,6 +899,78 @@ def parameters_table(
 
         return data
 
+    def update(
+        self,
+        exp_data: ExperimentData,
+        result_index: int = -1,
+        force_update: bool = False,
+        group: str = "default",
+    ):
+        """Update the calibrations form a result in the given experiment data.
+
+        This function allows users to update their calibrations from experiment data. Typically,
+        the value of the parameter to update is directly stored as the result of the fit. However,
+        for more complex cases, such as Rabi, the value of the parameter is extracted from the
+        fit result using the calibration extraction callable. Importantly, this method requires
+        the following keys to be present in the analysis result if no custom calibration extraction
+        function is given
+        - calibration_parameter: The name of the calibration parameter to update.
+        - qubits: The qubits to which the parameter belongs.
+        - calibration_schedule: The name of the schedule which is updated, this can be None (e.g.
+            for qubit frequencies).
+        - value: The value of the parameter which will enter the calibrations.
+        The ParameterKey formed by the values under (calibration_parameter, qubits,
+        calibration_schedule) must therefore be valid.
+
+        Args:
+            exp_data: An analysis result which contains either the value to update under the
+                key value or the information required by the calibration_extraction function
+                which will build the values to update.
+            result_index: The index of the result which defaults to 0.
+            force_update: If set to True then the calibrations will be updated even if the
+                quality of the result is "computer_bad".
+            group: The calibration group from which to draw the parameters.
+                If not specified this defaults to the 'default' group.
+
+        Raises:
+            CalibrationError: If the result does not have the required calibration keys. These
+                keys are "cal_parameter", "qubits", "cal_schedule", and "cal_value".
+        """
+        result = exp_data.analysis_result(result_index)
+
+        quality = result.get("quality", "computer_good")
+        if quality == "computer_bad" and not force_update:
+            return
```

Comment:
I feel substitution of "computer_good" is bit confusing and misleading. What we want to say here is quality is _defined_ but it says computer bad.
```suggestion
        if "quality" in result and result["quality"] == "computer_bad" and not force_update:
            return
```